packages/client/src/exampleTypes.js

```diff
@@ -138,6 +138,7 @@ export default F.stampKey('type', {
       page: 1,
       pageSize: 10,
       view: 'table',
+      hasResults: false,
```

Comment:
You shouldn't have to set this in the type definition because it's in the node defaults